cli/code_scanner.py

```diff
@@ -269,12 +270,22 @@ def exclude_irrelevant_files(context: click.Context, filenames: List[str]) -> Li
     return [filename for filename in filenames if _is_relevant_file_to_scan(scan_type, filename)]
 
 
-def exclude_irrelevant_detections(scan_type: str, scan_command_type: str, detections) -> List:
+def exclude_irrelevant_detections(scan_type: str, scan_command_type: str, min_severity: str, detections) -> List:
     relevant_detections = exclude_detections_by_exclusions_configuration(scan_type, detections)
     relevant_detections = exclude_detections_by_scan_command_type(scan_command_type, relevant_detections)
+    relevant_detections = exclude_detections_by_severity(scan_type, min_severity, relevant_detections)
+
     return relevant_detections
 
 
+def exclude_detections_by_severity(scan_type: str, min_severity: str, detections) -> List:
+    if scan_type != 'sca' or min_severity is None:
+        return detections
+
+    return [detection for detection in detections if
+            _is_valid_detection_severity(detection.detection_details.get('advisory_severity'), min_severity)]
```

Comment:
`_is_valid_detection_severity` - rename to `_does_severity_match_severity_threshold`